01week/rockPaperScissors.js

```diff
@@ -8,11 +8,31 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
 
+  // Write code here
+function rockPaperScissors(hand1, hand2){
+  if (hand1 === 'rock' && hand2 === 'rock') {
+    return "It's a tie!";
```

Comment:
The logic seems sound but you are repeating yourself a lot, think of a way reduce this